server/sonar-server/src/main/java/org/sonar/server/issue/filter/IssueFilterJsonWriter.java

```diff
@@ -21,31 +21,37 @@
 package org.sonar.server.issue.filter;
 
 import org.apache.commons.lang.StringUtils;
-import org.sonar.api.server.ServerSide;
 import org.sonar.api.utils.text.JsonWriter;
 import org.sonar.core.permission.GlobalPermissions;
 import org.sonar.db.issue.IssueFilterDto;
 import org.sonar.server.user.UserSession;
 
-@ServerSide
+import static com.google.common.base.Objects.firstNonNull;
+
 public class IssueFilterJsonWriter {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1118)